db/connection.js

```diff
@@ -9,4 +9,7 @@ if (!process.env.PGDATABASE) {
   throw new Error('PGDATABASE not set');
 }
 
+
+console.log("Connected to", ENV, "database")
```

Comment:
be careful leaving rogue console logs in your code if you are adding them, you needn't commit them :)